go/apps/dialogue/tests/test_definition.py

```diff
@@ -1,17 +1,19 @@
-from vumi.tests.helpers import VumiTestCase
+from twisted.internet.defer import inlineCallbacks
 
+from vumi.tests.helpers import VumiTestCase
 
 from go.vumitools.tests.helpers import VumiApiHelper
-from go.apps.dialogue.definition import ConversationDefinition
 from go.apps.dialogue.tests.dummy_polls import simple_poll
+from go.apps.dialogue.definition import ConversationDefinition
 
 
 class TestConversationDefinition(VumiTestCase):
+    @inlineCallbacks
     def setUp(self):
-        self.vumi_helper = self.add_helper(VumiApiHelper(is_sync=True))
```

Comment:
These are now async tests because conversation definitions are used in both sync and async contexts and the sync helper code pulls in Django.